.github/workflow/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This has been marked stale after 30 days with no activity. It will be closed in 5 days if there is no activity.'
+        stale-pr-message: 'Stale pull request message'
+        days-before-stale: 1
```

Comment:
Will you change this to 30 and 5 once your testing is done?

Reply:
done now. testing is on a different repo, so this should be what we really want.